backend/src/app.gateway.ts

```diff
@@ -26,9 +25,16 @@ export class AppGateway
   }
 
   @SubscribeMessage('message')
-  handleMessage(client: Socket, text: string): WsResponse<string> {
+  handleEvent(client: Socket, text: string): WsResponse<string> {
     const message = `Client with id: ${client.id} send a message: ${text}`;
     console.log(message);
     return { event: 'messageToClient', data: message };
   }
+
+  @SubscribeMessage('meeplePlacementMessage')
+  handleMessage(client: Socket, text: string): WsResponse<string> {
+    const message = `Client with id: ${client.id} put a meeple: ${text}`;
+    console.log(message);
+    return { event: 'messageToClientAfterMeeplePlacement', data: message };
+  }
```

Comment:
we should avoid repetition and make it more readable we may create ex. a helper class with methods like getBroadcastMessage (based on message type - meeplePlacementMessage, tilePlacementMessage). 
We can also add reusable function that broadcasts a message (function takes eventName and message itself)